src/Rules/Program.hs

```diff
@@ -37,8 +37,26 @@ buildProgram rs context@Context {..} = when (isProgram package) $ do
         -- We build only unwrapped binaries in inplace/lib/bin
         inplaceLibBinPath -/- programName context <.> exe %> \bin -> do
             binStage <- installStage
+            buildBinaryAndWrapperLib rs (context { stage = binStage }) bin
+
+        inplaceLibBinPath -/- programName context <.> "bin" <.> exe %> \bin -> do
```

Comment:
I actually don't know how it looks like on Windows, at least on Mac, the binary is `ghc-iserv.bin` and the wrapper is `ghc-iserv`. So I guess on Windows, the binary might be `ghc-iserv.bin.exe` and the wrapper is `ghc-iserv.exe`